shell/platform/windows/flutter_windows_view.h

```diff
@@ -206,6 +208,16 @@ class FlutterWindowsView : public WindowBindingHandlerDelegate {
 
   // Currently configured WindowBindingHandler for view.
   std::unique_ptr<flutter::WindowBindingHandler> binding_handler_;
+
+  // target size during resize event
+  size_t pending_width_, pending_height_;
+
+  // mutex and condition used during resize event
+  std::mutex resize_mutex_;
```

Comment:
Expand the comment to say that `resize_pending_`, `resize_target_width_` and `resize_target_height_` are also guarded by this mutex.